toolchain/check/convert.cpp

```diff
@@ -1180,6 +1188,20 @@ auto Convert(Context& context, SemIR::LocId loc_id, SemIR::InstId expr_id,
     return expr_id;
   }
 
+  // Defer the action if it's dependent.
```

Comment:
Might be good to have a TODO to restructure this code so that most of the code is in `PerformAction`, and we have a `HandleAction` here.